asserts/findwildcard.go

```diff
@@ -101,11 +117,67 @@ func findWildcardDescend(top, current string, descendantWithWildcard []string, f
 			return fmt.Errorf("findWildcard: invoked with malformed wildcard: %v", err)
 		}
 		if ok {
-			err = findWildcardDescend(top, filepath.Join(current, name), descendantWithWildcard[1:], foundCb)
+			err = findWildcardDescend(top, filepath.Join(current, name), descendantWithWildcard[1:], seqnum, foundCb)
 			if err != nil {
 				return err
 			}
 		}
 	}
 	return nil
 }
+
+func findWildcardSequence(top, current, seqWildcard string, descendantWithWildcard []string, seqnum int, foundCb func(relpath []string) error) error {
+	filter := func(i int) bool { return true }
+	if seqnum != -1 {
+		if seqWildcard == "#>" {
+			filter = func(i int) bool { return i > seqnum }
+		} else {
+			filter = func(i int) bool { return i < seqnum }
+		}
+	}
+
+	d, err := os.Open(current)
+	if os.IsNotExist(err) {
```

Comment:
Can you add a comment explaining what is the meaning of not-exists here and why it is not propagated?

Reply:
`current` can come straight from the query  (if it doesn't have wildcards for its level), in which case we just ignore the missing directory and the callback isn't called and the end result will be a not-found error at the higher level. This kind of code exist already in the other findWildcard* functions in the file

Reply:
Thanks for the comment